### PR TITLE
Create new panel J with PR curves from new models

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
# Create new panel J with PR curves from new models

In this short PR, a new PR curve is made for Panel J where we have the PR curves for both the actin and rest models (final and shuffled) for just training and testing data splits.

### CODE REVIEW

**Review by gwaybio — COMMENTED** [review]
Looks good! A couple general comments:

- The grid in the figure is a rectangle, it should be square (a step in both the y and x axis dimensions should be equivalent)
- Please add the full feature model performance here too

Of note, I'm not sure how Tim is using this figure (maybe he is recreating it?)

I'll approve once these are addressed!

File: 5.machine_learning/3.assess_generalizability/scripts/6.compare_models_plate3.py [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+        y_binary = le.transform(y)
+
+        # Predict class probabilities for morphology feature data
+        predicted_probs = model.predict_proba(X)
+
+        # Calculate the precision, recall data
+        precision, recall, threshold = precision_recall_curve(
+            y_binary, predicted_probs[:, -1]
+        )
+        threshold = np.append(threshold, np.nan)
```
Comment (gwaybio) [inline 5.machine_learning/3.assess_generalizability/scripts/6.compare_models_plate3.py:209]: Sorry if I'm missing something obvious, but are you also calculating AUPR somewhere?

**Review by gwaybio — APPROVED** [review]
Looks great! 🤩

File: 5.machine_learning/3.assess_generalizability/scripts/6.compare_models_plate3.py (hunk 1 above)
Comment (jenna-tomkinson) [reply 5.machine_learning/3.assess_generalizability/scripts/6.compare_models_plate3.py:209]: My bad! I did miss that, I have added it as the last code cell to the notebook, thanks!

**Comment by jenna-tomkinson** [comment]
@gwaybio Thanks for the review! I am merging now! 😄

**Pull request merged**